### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by Graycemuthui**
### Description
In this project I achieved the following:
- Build interactive console apps.
- Created an app.rb file that served as a console app entry-point which had the following methods:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain Person).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In the main.rb defined the entry point which had the following methods:
   - Present the user with a list of options to perform.
   - Lets users choose an option.
   - If needed, ask for parameters for the option.
   - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      puts 'No person in library'
+      return
+    end
+    @people.each do |person|
+      puts "[#{person.class}], Name:#{person.name}, ID:#{person.id}, Age:#{person.age}"
+    end
+  end
+
+  # Create student
+  def create_student
```
Comment (FrederickMih) [inline app.rb:48]: - [ ] Hi @Graycemuthui, fantastic job so far on implementing this method. Let's also make sure that permission should be granted to a student with an age less than 18 before being created. Once more, let's kindly watch this [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Graycemuthui , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- GitHub workflow has been followed :+1: 
- No Linter error :+1: 
- PR has a title and good summary :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      puts 'No person registered in the library'
+    else
+      puts 'Select a list from the following list by number'
+      @books.each_with_index do |book, index|
+        puts "#{index}) Title: #{book.title}, Author: #{book.author}"
+      end
+      rental_book = gets.chomp.to_i
+      puts 'Select a perspn from the following list by number'
+      @people.each_with_index do |person, index|
+        puts "#{index}) #{person.class} Name: #{person.name} ID:#{person.id} Age:#{person.age}"
```
Comment (ahmetbozaci) [inline app.rb:109]: ### OPTIONAL
_I wrote it as an optional but this is a requirement that missing the last reviewer. I highly recommend fixing it._

- [ ] I am getting an error when creating a rental.

Kindly fix it.
image

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@Graycemuthui  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: app.rb (hunk 2 above)
Comment (Graycemuthui) [reply app.rb:109]: Ok will work on it thanks

**Pull request merged**